backend/internal/server/apis/user/user.go

```diff
@@ -123,11 +135,22 @@ func createUserAccount(c *fiber.Ctx) error {
 	}
 
 	// response
-	createdUserResponse := createResponse{
-		Email:  user.Email,
-		UserId: int(u.ID),
-	}
+	// createdUserResponse := createResponse{
+	// 	Email:  user.Email,
+	// 	UserId: int(u.ID),
+	// }
+
+	return responder.OkWithData(c, u)
```

Comment:
I agree with the commented out response above, we should return the user including the ID but not the password

Reply:
changed back to original response since this was for debugging